app/src/gecko/java/org/mozilla/focus/web/GeckoViewPrompt.java

```diff
@@ -441,6 +441,7 @@ public void onItemClick(final AdapterView<?> parent, final View v,
                             return;
                         }
                     }
+                    Log.v("Onclick", adapter.getItem(position).toString());
```

Comment:
I guess this was just for debugging and doesn't need to land?

Reply:
Haha yep